lib/fog/google/requests/pubsub/create_subscription.rb

```diff
@@ -26,7 +26,7 @@ def create_subscription(subscription_name, topic, push_config = {}, ack_deadline
             "topic" => (topic.is_a?(Topic) ? topic.name : topic.to_s)
           }
 
-          unless push_config.empty?
+          if push_config && push_config.any?
```

Comment:
Had never used `.any?` before, but TIL!

```
2.4.0 :002 > nil && nil.any?
 => nil
2.4.0 :003 > {} && {}.any?
 => false
2.4.0 :004 > {a: 'b'} && {a: 'b'}.any?
 => true
```

Does not look like `tests/models/pubsub/` has any tests for this, nor does `tests/requests`. So I guess it's fine to just not have any for now.